src/test/scala/firrtlTests/ConstantPropagationTests.scala

```diff
@@ -9,7 +9,7 @@ import firrtl.passes._
 import firrtl.transforms._
 
 class ConstantPropagationSpec extends FirrtlFlatSpec {
-  val transforms = Seq(
+  def transforms = Seq(
```

Comment:
This seems spurious

Reply:
I don't think it is - the InferWidths has mutable state, so you don't want to share the same transform object; instead you want to create a new one.